bands/templates/band_admin/band_info.html

```diff
@@ -3,262 +3,115 @@
 {% block content %}
 
 <style type="text/css">
-    .col-sm-6 {
-        font-family: Courier;
-        padding-top: 8px;
-    }
+    .form-group { margin-bottom: 8px; }
 </style>
 
 <h2>Band Info for <a href="../bands/index?id={{ band.id }}">{{ band.group.name }}</a></h2>
 
-<br/> <br/>
 <form class="form-horizontal" role="form" method="post" action="band_info">
     {{ csrf_token() }}
     <input type="hidden" name="id" value="{{ band.id }}" />
 
-    {% if band.panel.name %}
-    <div class="row">
-        <label class="col-sm-2 control-label">Panel Name:</label>
-        <div class="col-sm-6">
-            {{ band.panel.name }}
-        </div>
-    </div>
-    {% endif %}
-
-    {% if band.panel.length %}
-    <div class="row">
-        <label class="col-sm-2 control-label">Panel Length:</label>
-        <div class="col-sm-6">
-            {{ band.panel.length }}
-        </div>
-    </div>
-    {% endif %}
-
-    {% if band.panel.desc %}
-    <div class="row">
-        <label class="col-sm-2 control-label">Panel Description:</label>
-        <div class="col-sm-6">
-            {{ band.panel.desc|linebreaksbr }}
-        </div>
-    </div>
-    {% endif %}
-
-    {% if band.panel.tech_needs %}
-    <div class="row">
-        <label class="col-sm-2 control-label">Panel Tech Needs:</label>
-        <div class="col-sm-6">
-            {{ band.panel.panel_tech_needs_labels|join(" / ") }}
-        </div>
-    </div>
-    {% endif %}
+    {{ macros.form_group(band.panel, 'name', label='Panel Name', is_readonly=True) }}
```

Comment:
I think I've read this right, but just to make sure: if there's no value for band.panel.name, this will just not display anything, right?